### PR TITLE
Release changelog for 2.24

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,67 @@
+snapcraft (2.24) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * meta: support classic confinment (#946)
```
Comment (kyrofa) [inline debian/changelog]: s/confinment/confinement/

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+  * sources: refactor deb source into module. (#963)
+  * sources: refactor local source into module. (#961)
+  * sources: refactor git source into module. (#964)
+  * sources: refactor mercurial source into module. (#965)
+  * sources: refactor rpm source into module. (#966)
+  * sources: refactor script source into module. (#967)
+  * sources: refactor subversion source into module. (#968)
+  * sources: refactor tar source into module. (#969)
+  * sources: refactor zip source into module. (#970)
+  * libraries: prefer in-snap libraries to system libraries. (#977)
```
Comment (kyrofa) [inline debian/changelog]: This was technically the pluginhandler, not libraries, but it involved libraries. I'm fine either way, just wanted to point it out.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+
+  [ Celso Providelo ]
+  * cli: implementing '[list-]registered' command (#890)
+  * store: support download and validate on branded stores. (#981)
+
+  [ Matthew Aguirre ]
+  * maven plugin: implement get_build_properties() (#974)
+  * gradle plugin: implement get_build_properties()  (#975)
```
Comment (kyrofa) [inline debian/changelog]: Extra space here between the title and PR reference.

File: debian/changelog (hunk 2 above)
Comment (sergiusens) [inline debian/changelog]: I originally had pluginhandler, but meh

File: debian/changelog (hunk 3 above)
Comment (sergiusens) [inline debian/changelog]: I'll ignore this one for now, no need to trigger adt again for this

**Pull request merged**